drivers/SmartThings/matter-hrap/src/WiFiNetworkManagement/init.lua

```diff
@@ -0,0 +1,125 @@
+WiFiNetworkManagement.server.commands = WiFiNetworkManagementServerCommands:set_parent_cluster(WiFiNetworkManagement)
+WiFiNetworkManagement.client.commands = WiFiNetworkManagementClientCommands:set_parent_cluster(WiFiNetworkManagement)
+WiFiNetworkManagement.types = WiFiNetworkManagementTypes
+
+function WiFiNetworkManagement:get_attribute_by_id(attr_id)
+  local attr_id_map = {
+    [0x0000] = "Ssid",
+    [0x0001] = "PassphraseSurrogate",
+    [0xFFF9] = "AcceptedCommandList",
+    [0xFFFA] = "EventList",
+    [0xFFFB] = "AttributeList",
+  }
+  local attr_name = attr_id_map[attr_id]
+  if attr_name ~= nil then
+    return self.attributes[attr_name]
+  end
+  return nil
+end
+
+function WiFiNetworkManagement:get_server_command_by_id(command_id)
+  local server_id_map = {
+    [0x0000] = "NetworkPassphraseRequest",
+  }
+  if server_id_map[command_id] ~= nil then
+    return self.server.commands[server_id_map[command_id]]
+  end
+  return nil
+end
+
+function WiFiNetworkManagement:get_client_command_by_id(command_id)
+  local client_id_map = {
+    [0x0001] = "NetworkPassphraseResponse",
+  }
+  if client_id_map[command_id] ~= nil then
+    return self.client.commands[client_id_map[command_id]]
+  end
+  return nil
+end
+
+WiFiNetworkManagement.attribute_direction_map = {
+  ["Ssid"] = "server",
+  ["PassphraseSurrogate"] = "server",
+  ["AcceptedCommandList"] = "server",
+  ["EventList"] = "server",
+  ["AttributeList"] = "server",
+}
+
+do
+  local has_aliases, aliases = pcall(require, "st.matter.clusters.aliases.WiFiNetworkManagement.server.attributes")
+  if has_aliases then
+    for alias, _ in pairs(aliases) do
+      WiFiNetworkManagement.attribute_direction_map[alias] = "server"
+    end
+  end
+end
+
+WiFiNetworkManagement.command_direction_map = {
+  ["NetworkPassphraseRequest"] = "server",
+  ["NetworkPassphraseResponse"] = "client",
+}
+
+do
+  local has_aliases, aliases = pcall(require, "st.matter.clusters.aliases.WiFiNetworkManagement.server.commands")
+  if has_aliases then
+    for alias, _ in pairs(aliases) do
+      WiFiNetworkManagement.command_direction_map[alias] = "server"
+    end
+  end
+end
+
+do
+  local has_aliases, aliases = pcall(require, "st.matter.clusters.aliases.WiFiNetworkManagement.client.commands")
+  if has_aliases then
+    for alias, _ in pairs(aliases) do
+      WiFiNetworkManagement.command_direction_map[alias] = "client"
+    end
+  end
+end
+
+local attribute_helper_mt = {}
+attribute_helper_mt.__index = function(self, key)
+  local direction = WiFiNetworkManagement.attribute_direction_map[key]
+  if direction == nil then
+    error(string.format("Referenced unknown attribute %s on cluster %s", key, WiFiNetworkManagement.NAME))
+  end
+  return WiFiNetworkManagement[direction].attributes[key]
+end
+WiFiNetworkManagement.attributes = {}
+setmetatable(WiFiNetworkManagement.attributes, attribute_helper_mt)
+
+local command_helper_mt = {}
```

Comment:
Similar to above, code referencing the commands and events which are not used in the embedded cluster definition can be removed here and throughout the other embedded cluster files to reduce driver package size.